### ISSUE TITLE
Block validation failed - does this with each block (refers to min/max 615/617)

### ISSUE THREAD

**Issue by ericjarvies** [theQRL/QRL#494]
The following occurs for each block it attempts to fetch (min/max **615**/**617**);

```
2017-10-24 18:29:09,581 |synced| INFO : Block validation failed
2017-10-24 18:29:09,581 |synced| INFO : Block #649
2017-10-24 18:29:09,581 |synced| INFO : Stake_selector b'Q0aafda6e082e072533194acebd0f6a82c38cfdca8aa562a317b4275676c93b1cb6988c8f'
2017-10-24 18:29:09,582 |synced| INFO : >>>Received block from 104.237.3.185:9000 650 b'Q05f82cfb3bec7ee85106022baf6e33ea78f257298bbc078275aa6a60c861887036ff6f69'
2017-10-24 18:29:09,582 |synced| ERROR : get_block_n blocknum not found in blocks 649
2017-10-24 18:29:09,582 |synced| ERROR : Min block num **615**
2017-10-24 18:29:09,582 |synced| ERROR : Max block num **617**
2017-10-24 18:29:09,583 |synced| ERROR : Traceback (most recent call last):
  File "/Users/propertydb/homebrew/lib/python3.6/site-packages/qrl/core/block.py", line 101, in validate_block
    if not self.blockheader.validate(last_block.blockheader):
AttributeError: 'NoneType' object has no attribute 'blockheader'
```

**Comment by ericjarvies** [comment]
Here is my [OS X] log file;
[qrl.log.zip](https://github.com/theQRL/QRL/files/1412893/qrl.log.zip)

**Comment by cyyber** [comment]
The above issue has been fixed in PR #495. I will keep this issue open for few more hours for further confirmation by testing on other nodes.

**Comment by cyyber** [comment]
Nodes working fine after PR #495.
